api/src/test/java/com/netflix/iceberg/expressions/TestEvaluatior.java

```diff
@@ -145,6 +146,18 @@ public void testNot() {
     Assert.assertTrue("not(8 == 7) => false", evaluator.eval(TestHelpers.Row.of(8)));
   }
 
+  @Test
+  public void testCaseInsensitiveNot() {
+    Evaluator evaluator = new Evaluator(STRUCT, not(equal("X", 7)), false);
+    Assert.assertFalse("not(7 == 7) => false", evaluator.eval(TestHelpers.Row.of(7)));
+    Assert.assertTrue("not(8 == 7) => false", evaluator.eval(TestHelpers.Row.of(8)));
+  }
+
+  @Test(expected = ValidationException.class)
```

Comment:
Minor: `TestHelpers` has an `assertThrows` assertion that can validate the exception message. It also allows you to reuse code more easily.

Reply:
Fixing so that all tests added that did `@Test(expected = ValidationException.class)` will now use `TestHelpers#assertThrows`.